sdk/python/kfp/compiler/_op_to_template.py

```diff
@@ -288,6 +289,15 @@ def _op_to_template(op: BaseOp):
         template['volumes'] = [convert_k8s_obj_to_json(volume) for volume in processed_op.volumes]
         template['volumes'].sort(key=lambda x: x['name'])
 
+    # resource requests at runtime
+    if isinstance(op, dsl.ContainerOp) and (processed_op.cpu_request or processed_op.memory_request):
+        podSpecPatch = {'containers':[{'name':'main', 'resources':{'limits':{}}}]}
+        if processed_op.cpu_request: 
+            podSpecPatch['containers'][0]["resources"]["limits"]["cpu"] = processed_op.cpu_request
```

Comment:
nit: use consistent quotes (single quotes seem to be the dominant choice within this file and probably within the codebase as well).